tests/json-fixtures/test_transactions.py

```diff
@@ -5,6 +5,7 @@
 import rlp
 
 from eth.exceptions import UnrecognizedTransactionType
+from eth.vm.forks.shanghai.transactions import ShanghaiTransactionBuilder
```

Comment:
For style matching, it looks like this should go down below `ParisTransactionBuilder`. But it'll probably be resolved when we add `isort` anyway.

Reply:
I'd prefer to wait until we add it to make these sorts of changes